src/aleph/sdk/chains/common.py

```diff
@@ -143,22 +148,145 @@ async def decrypt(self, content: bytes) -> bytes:
             raise NotImplementedError
 
 
-# Start of the ugly stuff
 def generate_key() -> bytes:
+    """
+    Generate a new private key.
+
+    Returns:
+    bytes: The generated private key as bytes.
+    """
+
     privkey = PrivateKey()
     return privkey.secret
 
 
+def create_key() -> bytes:
+    """
+    Create or import a private key.
+
+    This function allows the user to either import an existing private key
+    or generate a new one. If the user chooses to import a key, they can
+    enter a private key in hexadecimal format or a passphrase.
+
+    Returns:
+    bytes: The private key as bytes. 7e0c27fff7e434ec5aa47127e7bcdce81c4eba6f3ce980f425b60b1cd019a947
+    """
+    if Prompt.ask("Import an existing wallet", choices=["y", "n"], default="n") == "y":
+        data = Prompt.ask("Enter your private key or passphrase")
+        # private key
+        if data.startswith("0x") and len(data) == 66:
+            return bytes.fromhex(data[2:])
+        elif len(data) == 64:
+            return bytes.fromhex(data)
+        # passphrase
+        elif len(data.split()) in [12, 24]:
+            w3 = Web3()
+            w3.eth.account.enable_unaudited_hdwallet_features()
+            return w3.eth.account.from_mnemonic(data.strip()).key
+        raise ValueError("Invalid private key or passphrase")
+    return generate_key()
```

Comment:
Make it explicit that this is not called as a fallback.
```suggestion
    else:
        return generate_key()
```